src/pages/signin/LoginForm.js

```diff
@@ -231,7 +231,14 @@ function LoginForm(props) {
                             containerStyles={[styles.mh0]}
                         />
                         <View style={[getSignInWithStyles()]}>
-                            <Text style={[styles.textLabelSupporting, styles.textAlignCenter, styles.mb3, styles.mt2]}>{props.translate('common.signInWith')}</Text>
+                            <Text
+                                accessibilityElementsHidden
+                                importantForAccessibility="no-hide-descendants"
```

Comment:
Did you try out aria-hidden? if that works for everything that would be really nice.

I would also add a comment explaining why we're doing this, e.g.,

```
{/** Hide "Sign in with:" text from screen readers, as the entire phrase ("Sign in with Apple") should be contained within the labels for each button's label **/}
<Text ...
```

Reply:
I did try the ariaHidden, but can double check.